### ISSUE TITLE
Is this blockchain abandoned?

### ISSUE THREAD

**Issue by kaber2** [Fantom-foundation/go-opera#546]
Given that the master branch hasn't received any commits in 2 years and all other branches are listed as stale, the use of a very old geth version with known security issues (f.i. p2p ping goroutine exhaustion) and the general lack of other activity, I would like to know if this project has been abandoned.

**Comment by happypsyduck** [comment]
It seems that the develop branch of this repository has more updates to Fantom Opera (last update was 7 months ago). The owner of the repository (@andrecronje ) doesn't seem to be approving commits to master branch lately.

**Comment by uprendis** [comment]
Hello All,
Good day to you!
Yes, the development has moved to the new repo: https://github.com/Fantom-foundation/Sonic
This repo will get archived eventually

**Comment by kaber2** [comment]
Thank you for the information. This leaves a few questions though. Is this new fork compatible with opera, and will it be running on the same chain or a new one? Or differently asked, is there anything a Fantom node operator needs to do?

**Comment by uprendis** [comment]
Yeah, it's compatible but will require a resync (not compatible with current datadir). The upgrade is in progress. Eventually there will be a new chain (called Sonic) too as was voted by the governance (in proposal 41)

**Comment by zhangxf55** [comment]
> Yeah, it's compatible but will require a resync (not compatible with current datadir). The upgrade is in progress. Eventually there will be a new chain (called Sonic) too as was voted by the governance (in proposal 41)

I have tried and failed to download the snapshot from https://snapshot.fantom.network/files/pruned/ and run the node with go-opera. I still have some questions:
1. After the upgrade, would the chainId(0xfa) be different and the history transaction be abandoned?
2. If not, can the snapshot still be used to run a rpc node or I have to sync the data from the very begining?
3. I read the doc from https://docs.fantom.foundation/node/tutorials/opera-client/run-an-api-node and tried to run a node, it is not updated and where is the latest documentation?
4. Can sonic v1.2.1-c be used in the fantom mainnet now?
5. Is it a bug: https://github.com/Fantom-foundation/go-opera/issues/545 and will it be fixed? Or I just need to jump to sonic instead?
